### PR TITLE
feat(ids): Add application system urls. Add localhost to dev.

### PR DESCRIPTION
**Pull request by valurefugl**
## What

Add urls for application system to allowed redirect urls. Add localhost urls to allowed urls on dev.

## Why

So the application system can integrate with user profile collection on the IDS.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/services/auth/ids-api/infra/identity-server.ts [hunk 1]
```diff
@@ -84,9 +84,10 @@ export const serviceSetup = (services: {
         prod: 'https://cdn.contentful.com',
       },
       Application__AllowedRedirectUris: {
-        dev: 'https://beta.dev01.devland.is/minarsidur',
-        staging: 'https://beta.staging01.devland.is/minarsidur',
-        prod: 'https://island.is/minarsidur',
+        dev: 'https://beta.dev01.devland.is/minarsidur,https://beta.dev01.devland.is/umsoknir,http://localhost:4200/minarsidur,http://localhost:4242/umsoknir',
```
Comment (robertaandersen) [inline apps/services/auth/ids-api/infra/identity-server.ts:87]: Formatting

File: apps/services/auth/ids-api/infra/identity-server.ts [hunk 2]
```diff
@@ -84,9 +84,10 @@ export const serviceSetup = (services: {
...
+        staging:
+          'https://beta.staging01.devland.is/minarsidur,https://beta.staging01.devland.is/umsoknir',
```
Comment (robertaandersen) [inline apps/services/auth/ids-api/infra/identity-server.ts:89]: Formatting (remove newline)

**Review by brynjarorng — APPROVED** [review]
LGTM

**Review by robertaandersen — APPROVED** [review]
LGTM

**Pull request merged**